archive/components/embedding/factory.py

```
@@ -72,7 +101,9 @@ def create_embedder(embedder_type: str, config: Optional[Dict[str, Any]] = None)
     if embedder_type == "jina_v4":
         # Import here to avoid circular imports
         from src.jina_v4.jina_processor import JinaV4Processor
-        return JinaV4Processor(config)
+        import asyncio
```

Flake8 (7.2.0)

[error] 'asyncio' imported but unused

(F401)